ldms/src/core/ldms_xprt.c

```diff
@@ -795,7 +802,6 @@ static void process_dir_request(struct ldms_xprt *x, struct ldms_request *req)
 				goto out;
 			}
 			reply->hdr.len = htonl(last_cnt + hdrlen);
-			reply->dir.json_data_len = htonl(last_cnt);
```

Comment:
We should set this value to -1 or something and add doc to indicate that it is deprecated...

Reply:
I added the line back so that the path in 4.3.4 is compatible with 4.3.3. __process_dir_reply() in v4.3.3 still uses reply->dir.json_data_len.